tests/bin/setup-node.sh

```diff
@@ -9,23 +9,45 @@ set -eo pipefail
 apt-get install -y apt-transport-https
 
 # install docker
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
-sh -c "echo deb https://get.docker.com/ubuntu docker main > /etc/apt/sources.list.d/docker.list"
-apt-get update && apt-get install -yq lxc-docker-1.5.0
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
+            --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
+echo deb https://get.docker.com/ubuntu docker main > /etc/apt/sources.list.d/docker.list
+apt-get update
+apt-get install -yq lxc-docker-1.5.0
+
+# install extra extensions (AUFS)
+sudo apt-get -y install "linux-image-extra-$(uname -r)"
```

Comment:
Is there a reason we're using aufs on the tests? We default to overlayfs in shipped Deis, so ideally we're using that here as well. I recall some history with this...